sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -266,6 +266,7 @@ export interface AssessmentModel {
 }
 
 export interface CalculatedAssessmentModel {
+  calculatedDataRemainingScholarshipsBursariesLimit: any;
```

Comment:
Please replace `any` with `number`.
`any` should not be used unless under a very special case.

Reply:
This one was missed or the code was not pushed, can you please check?